docs/docs-content/getting-started/vmware/vmware.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_label: "Deploy a Cluster to VMware"
+title: "Deploy a Cluster to VMware"
+description: "Spectro Cloud Getting Started with VMware"
+hide_table_of_contents: false
+sidebar_custom_props:
+  icon: ""
+tags: ["getting-started", "vmware"]
+---
+
+Palette supports using [VMware](https://www.vmware.com) as a data center provider. You can deploy and manage
+[Host Clusters](../../glossary-all.md#host-cluster) on VMware virtual machines.
```

Comment:
```suggestion
Palette supports integration with [VMware](https://www.vmware.com). You can deploy and manage
[Host Clusters](../../glossary-all.md#host-cluster) on VMware.
```
I think I would follow the same pattern we used for the other providers, but its up to you :)